Extras~/ReleaseNotes.txt

```diff
@@ -6,6 +6,7 @@
 - Bugfix (1256530) - disallow multiple components where appropriate
 - Bugfix: BlendList camera was incorrectly holding 0-length camera cuts
 - Bugfix (1174993) - CM Brain logo was not added to Hierarchy next to Main Camera after adding vcam for the first time after importing CM. 
+- UI update - Moved Cinemachine menu to GameObject Create menu and Right Click context menu for Hierachy window.
```

Comment:
This entire file can now be deleted, since it was only ever used in the About window.